test/test_plugin_helper.rb

```diff
@@ -10,6 +10,7 @@
 
 begin
   require 'pry-rescue/minitest'
+  require 'pry-byebug'
```

Comment:
Are we wanting to add this?

Reply:
it's for proper debugging of the tests. If the gem is not loaded, it will bail out. No harm done.

Reply:
👍